images/installer/README_INVENTORY_GENERATOR.md

```diff
@@ -0,0 +1,90 @@
+Dynamic Inventory Generation
+============================
+
+Script within the openshift-ansible image that can dynamically
+generate an Ansible inventory file from an existing cluster.
+
+### Configure
```

Comment:
We are missing the second level of headers, we drop from the first to the third. We could s/###/##/g.